README.md

```diff
@@ -1,6 +1,76 @@
+# [`viamrtsp` module](https://app.viam.com/module/erh/viamrtsp)
+
+This module implements the [`"rdk:component:camera"` API](https://docs.viam.com/components/camera/) for real-time streaming protocol (RTSP) enabled cameras.
+Four models are provided:
+* `erh:viamrtsp:rtsp` - Codec agnostic. Will auto detect the codec of the `rtsp_address`.
+* `erh:viamrtsp:rtsp-h264` - Only supports H264 codec.
+* `erh:viamrtsp:rtsp-h265` - Only supports H265 codec.
+* `erh:viamrtsp:rtsp-mjpeg` - Only supports M-JPEG codec.
+
+## Configure your `viamrtsp` camera
+
+Navigate to the [**CONFIGURE** tab](https://docs.viam.com/build/configure/) of your [machine](https://docs.viam.com/fleet/machines/) in the [Viam app](https://app.viam.com/).
+[Add the camera component to your machine](https://docs.viam.com/build/configure/#components), searching for `viamrtsp` and selecting your desired model.
+
+Copy and paste the following attributes template into the resulting component's attribute panel:
+
+```
+{
+   "rtp_passthrough": true,
+   "rtsp_address": "rtsp://foo:bar@192.168.10.10:554/stream"
+}
+```
+
+Edit the attributes as applicable.
+
+### Attributes
+
+The following attributes are available for all models of `viamrtsp` cameras:
+
+| Name    | Type   | Inclusion    | Description |
+| ------- | ------ | ------------ | ----------- |
+| `rtp_passthrough` | bool | **Required** | RTP passthrough mode (which improves video streaming efficiency) is supported with the H264 codec if this attribute is set to `true`. |
+| `rtsp_address` | string | **Required** | The RTSP address where the camera streams. |
+| `intrinsic_parameters` | object | Optional | The intrinsic parameters of the camera used to do 2D <-> 3D projections: <ul> <li> `width_px`: The expected width of the aligned image in pixels. </li> <li> `height_px`: The expected height of the aligned image in pixels. </li> <li> `fx`: The image center x point. </li> <li> `fy`: The image center y point. </li> <li> `ppx`: The image focal x. </li> <li> `ppy`: The image focal y. </li> </ul> |
```

Comment:
can remove these from here if we don't want to surface?

Reply:
I have not used `intrinsic_parameters` nor `distortion_parameters` with this camera model & can't speak to them. I personally think it would be fine to not document them as I have not done any testing with them.

Reply:
Ok!